packages/Bio/src/tests/detectors-benchmark-tests.ts

```diff
@@ -23,35 +21,35 @@ category('detectorsBenchmark', () => {
   // -- fasta --
 
   test('fastaDnaShorts50Few50', async () => {
-      const et: number = await detectMacromoleculeBenchmark(10, NOTATION.FASTA, ALPHABET.DNA, 50, 50);
-    },
-    {skipReason: '#1192'});
+    const _et: number = await detectMacromoleculeBenchmark(10, NOTATION.FASTA, ALPHABET.DNA, 50, 50);
```

Comment:
This variable seem to be never used and doesn't provide any info. I guess we can remove it.